xprof/btx_aggreg_model.yaml

```diff
@@ -0,0 +1,135 @@
+:stream_classes:
+  - :name: interval_tally
+    :event_common_context_field_class:
+      :type: structure
+      :members:
+        - :name: hostname
+          :field_class:
+            :type: string
+            :cast_type: const char*
+        - :name: vpid
+          :field_class:
+            :type: integer_signed
+            :field_value_range: 64
+            :cast_type: int64_t
+        - :name: vtid
+          :field_class:
+            :type: integer_unsigned
+            :field_value_range: 64
+            :cast_type: uint64_t
+        - :name: name
+          :field_class:
+            :type: string
+            :cast_type: const char*
+        - :name: min
+          :field_class:
+            :type: integer_unsigned
+            :field_value_range: 64
+            :cast_type: uint64_t
+        - :name: max
+          :field_class:
+            :type: integer_unsigned
+            :field_value_range: 64
+            :cast_type: uint64_t
+        - :name: total
+          :field_class:
+            :type: integer_unsigned
+            :field_value_range: 64
+            :cast_type: uint64_t
+        - :name: count
+          :field_class:
+            :type: integer_unsigned
+            :field_value_range: 64
+            :cast_type: uint64_t
+    :event_classes:
+      - :name: tally_host
+        :payload_field_class:
+          :type: structure
+          :members:
+            - :name: backend
+              :field_class:
+                :type: integer_unsigned
+                :field_value_range: 64
+                :cast_type: uint64_t
+            - :name: err_count
+              :field_class:
+                :type: integer_unsigned
+                :field_value_range: 64
+                :cast_type: uint64_t
+      - :name: tally_device
+        :payload_field_class:
+          :type: structure
+          :members:
+            - :name: did
+              :field_class:
+                :type: integer_unsigned
+                :field_value_range: 64
+                :cast_type: uint64_t
+            - :name: sdid
+              :field_class:
+                :type: integer_unsigned
+                :field_value_range: 64
+                :cast_type: uint64_t
+            - :name: metadata
+              :field_class:
+                :type: string
+                :cast_type: const char*
+      - :name: tally_traffic
+        :payload_field_class:
+          :type: structure
+          :members:
+            - :name: backend
+              :field_class:
+                :type: integer_unsigned
+                :field_value_range: 64
+                :cast_type: uint64_t
+  - :name: interval
```

Comment:
Why do we need this stream class on this file?

Reply:
Require to be able to read the lttng::device name

Reply:
Exacly. We can refactor everything  cleanly when we move the interval to metababel